stylesheets/styles.css

```diff
@@ -63,7 +63,7 @@ body, html {
     border: .1em solid gray;
     font-size: 1.5em;
     border-radius: 0.2em;
-
+    display: flex;
     /* Temporary background color to see layout */
```

Comment:
A curiosity 🤔:

- Maybe we could remove the duplicate `display: flex` attribute for .listing? It looks like the display has already been set on line 58.